src/account/account.cairo

```diff
@@ -0,0 +1,218 @@
+use core::array::ArrayTrait;
+use core::array::SpanTrait;
+use starknet::account::Call;
+use starknet::ContractAddress;
+
+
+
+// #[starknet::interface]
+// trait IsupportInterface<TContractState>{
+//     fn register_interface(self:@TContractState, interface_id: felt252) -> bool;
+//     fn deresgister_interface(self:@TContractState, interface_id:felt252) -> bool;
+// }
+
+const QUERY_VERSION: felt252 = 340282366920938463463374607431768211457;
+const TRANSACTION_VERSION: felt252 = 1;
```

Comment:
Let's remove these consts for now to simplify our implementation and come up with a POC that works. We could add these later on, if we deem necessary.

PS: remember to edit the `__execute__` function too as it makes use of these consts